uwrt_mars_rover_hw/drivers/roboteq_driver/mbs/roboteq_control_mode_switching.mbs

```diff
@@ -0,0 +1,18 @@
+print("Starting roboteq drivetrain control mode switching ... \r")
+current_control_mode_1 = getvalue(_MMOD, 1)
+current_control_mode_2 = getvalue(_MMOD, 2)
+
+top: 
+desired_control_mode = getvalue(_VAR, 9) ' Read desired control mode from ROS
+
+if(current_control_mode_1 = current_control_mode_1 And desired_control_mode <> current_control_mode_1) Then
+	current_control_mode_1 = setconfig(_MMOD, 1, desired_control_mode) ' switch control modes
+	current_control_mode_2 = setconfig(_MMOD, 2, desired_control_mode) 
+	
+	if(current_control_mode_1 = current_control_mode_2) Then
+		setvalue(_VAR, 1, current_control_mode_1) ' set TPDO to current control mode
+		setvalue(_VAR, 2, current_control_mode_2) 
+	endif
+end if
+wait(100) ' TODO: TPDO set to 100ms

```

Comment:
I think TPDOs should report at 100hz, so 10ms

Reply:
![image](https://user-images.githubusercontent.com/56097508/80742555-8f098400-8ae9-11ea-9315-8e16e4cd5d34.png)

Reply:
i more meant that we should try to be at 100hz. could u change the values in the profile xml. u can load the file into roborun.

Reply:
could u do this

Reply:
I changed the profile.xml to 10ms and try to load the config using this
`wait(GetConfig(_CTPS, 1)) ' TPDO set to 10ms`

The manual only lists a setconfig
![image](https://user-images.githubusercontent.com/56097508/87316314-9c80b880-c4f3-11ea-8feb-c3b71aae4ca9.png)